chargebee/src/main/java/com/chargebee/android/billingservice/CBPurchase.kt

```diff
@@ -240,5 +231,10 @@ object CBPurchase {
         if (arr.size==1) list.add("Standard")
         return list.toTypedArray()
     }
-
+    private fun shareInstance(context: Context): BillingClientManager {
+        if (billingClientManager == null) {
+            billingClientManager = BillingClientManager(context)
+        }
+        return billingClientManager as BillingClientManager
+    }
```

Comment:
Minor: Can `billingClientManager` be a lateinit? And should we use `shareInstance` in the `retrieveProducts` method?